### PR TITLE
Adding MASVS v2 Compliance

### PR DESCRIPTION
**Pull request by poseidontor**
Added Enum for MASVS v2 Compliance and updated report template to include MASVS v2 compliance mappings.

### CODE REVIEW

File: ak_vendor/report/report.py [hunk 1]
```diff
@@ -365,6 +372,10 @@ def create_asvs(cls, code: str, title: str) -> ASVS:
     def create_mstg(cls, code: str, title: str) -> MSTG:
         return MSTG(code=code, title=title)
 
+    @classmethod
+    def create_masvs(cls, code: str, title: str) -> MASVS:
```
Comment (rajan262) [inline ak_vendor/report/report.py:376]: The type annotation should be quoted. This is not a requirement per say but it's a good practice. Change the line to `def create_masvs(cls, code: str, title: str) -> "MASVS":`

File: ak_vendor/templates/report_template.html [hunk 2]
```diff
@@ -946,7 +946,7 @@ <h2 class="analysis__cvss__scorevalue" style="color: {{ analysis.risk.value_colo
             </table>
           </section>
 
-          {% if analysis.regulatory.owasp or analysis.regulatory.cwe or analysis.regulatory.mstg or analysis.regulatory.asvs or analysis.regulatory.pcidss or analysis.regulatory.hipaa or analysis.regulatory.gdpr %}
+          {% if analysis.regulatory.owasp or analysis.regulatory.cwe or analysis.regulatory.mstg or analysis.regulatory.masvs or analysis.regulatory.asvs or analysis.regulatory.pcidss or analysis.regulatory.hipaa or analysis.regulatory.gdpr %}
```
Comment (rajan262) [inline ak_vendor/templates/report_template.html:949]: @Praseetha-KR @subho007 Should we remove this condition? Is it even possible for any testcase to have no compliance in this long list of compliances? 😆

File: ak_vendor/report/report.py (hunk 1 above)
Comment (poseidontor) [reply ak_vendor/report/report.py]: Made the changes.

File: ak_vendor/report/report.py [hunk 3]
```diff
@@ -400,6 +411,10 @@ def add_mstg(self, mstg: MSTG) -> List[MSTG]:
         self.mstg.append(mstg)
         return self.mstg
 
+    def add_masvs(self, masvs: MASVS) -> List[MASVS]:
```
[the remark this answers is not in the document]
Comment (poseidontor) [reply ak_vendor/report/report.py]: Made the changes.

File: ak_vendor/templates/report_template.html (hunk 2 above)
Comment (Praseetha-KR) [reply ak_vendor/templates/report_template.html:949]: @rajan262 on a second thought, let's go ahead without removing it. If we have to make all compliances optional, then this if-condition will become necessary. Let's wait until the compliance based reports are implemented then we might have a strong reasoning.

**Review by rajan262 — APPROVED** [review]
Looks good

**Pull request merged**